airflow/auth/managers/fab/fab_auth_manager.py

```diff
@@ -61,17 +61,27 @@ def get_cli_commands() -> list[CLICommand]:
             SYNC_PERM_COMMAND,  # not in a command group
         ]
 
+    def get_user_display_name(self) -> str:
+        """Return the user's display name associated to the user in session."""
+        user = self.get_user()
+        first_name = user.first_name or ""
+        last_name = user.last_name or ""
+        return f"{first_name} {last_name}".strip()
```

Comment:
```suggestion
        first_name = user.first_name or ""
        last_name = user.last_name or ""
        return f"{first_name.strip()} {last_name.strip()}"
```

Reply:
I like the current code better because if no first name and last name are set, then it returns a empty string which is Falsy. On the UI, the `get_user_display_name` is Falsy, we do something specific. This use case is real and happened not a long time ago. See #33586

Reply:
Maybe `f"{first_name.strip()} {last_name.strip()}".strip()?` Ugly, but it works to accomplish both goals.

Reply:
Settled on:

```py
first_name = user.first_name.strip() or ""
last_name = user.last_name.strip() or ""
return f"{first_name} {last_name}".strip()
```